aiomysql/connection.py

```diff
@@ -166,6 +175,17 @@ def __init__(self, host="localhost", user=None, password="",
         self._server_auth_plugin = ""
         self._auth_plugin_used = ""
 
+        # TODO somehow import version from __init__.py
+        self._connect_attrs = {
+            '_client_name': 'aiomysql',
+            '_pid': str(os.getpid()),
```

Comment:
is this something new mysql requires for connection?

Reply:
This is where it gets weird. Yes, but I also couldn't seem to connect to versions like 5.6.36 without it either :/ which got me thinking, did PyMySQL change some variable somewhere that were using telling the client we can send client info, and yep: https://github.com/PyMySQL/PyMySQL/commit/935afc8130534852c5e2c1396564f8262121bc2f#diff-2e0519e4fb7a37ffc11d27d760956f00

We should probably only send this if CONNECT_ATTRS is in the capabilities list, but we also don't change that so it should be fine.